src/main/java/pippi/task/Deadline.java

```diff
@@ -0,0 +1,29 @@
+package pippi.task;
+
+import pippi.parser.DateFormatter;
+
+import java.time.LocalDate;
+public class Deadline extends Task {
+    private LocalDate due;
+    public Deadline(String desc, LocalDate due) {
+        super(desc);
+        this.due = due;
+    }
+
+    /**
+     * Returns the String representation of a deadline task to the UI
+     * @return Deadline string representation

```

Comment:
I think there should be a blank line between description and parameter?
I didn't comment this on the rest as it would be too much.
<img width="394" alt="image" src="https://github.com/nus-cs2103-AY2324S1/ip/assets/107026005/a6512401-c842-4398-ba30-199ec6480e55">